### PR TITLE
add clamp api, test=develop

### PR DESCRIPTION
**Pull request by jerrywgz**
image
image

### CODE REVIEW

**Review by XiaoguangHu01 — DISMISSED** [review]

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -1255,3 +1256,89 @@ def log1p(x, out=None, name=None):
         out = helper.create_variable_for_type_inference(dtype)
     helper.append_op(type="log1p", inputs={"X": x}, outputs={"Out": out})
     return out
+
+
+def clamp(input, min=None, max=None, output=None, name=None):
+    """
+    **clampe layer**
+
+    This operator clamp all elements in input into the range [ min, max ] and return
```
Comment (Boyan-Liu) [inline python/paddle/tensor/math.py]: clamps

File: python/paddle/tensor/math.py [hunk 2]
```diff
@@ -1255,3 +1256,89 @@ def log1p(x, out=None, name=None):
...
+    a resulting tensori as the following equation:
```
Comment (Boyan-Liu) [inline python/paddle/tensor/math.py]: tensor

File: python/paddle/tensor/math.py (hunk 1 above)
Comment (jerrywgz) [reply python/paddle/tensor/math.py]: Done

File: python/paddle/tensor/math.py (hunk 2 above)
Comment (jerrywgz) [reply python/paddle/tensor/math.py]: Done

**Review by Boyan-Liu — DISMISSED** [review]

**Review by XiaoguangHu01 — DISMISSED** [review]

**Review by heavengate — APPROVED** [review]

**Review by Boyan-Liu — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LG API

**Pull request merged**